google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -589,6 +589,15 @@ public Builder setSystemExecutorProvider(ExecutorProvider executorProvider) {
       return this;
     }
 
+    /**
+     * Sets the number of pullers used to pull messages from the subscription. Defaults to the
+     * number of available processors.
+     */
+    public Builder setNumPullers(int numPullers) {
```

Comment:
I think the first is more meaningful from the user's perspective - they don't have to know that pubsub is implemented with streams exactly. In Java, length is a less important metric than other languages :-P